### PR TITLE
new contributors.md format

### PR DESCRIPTION
**Pull request by mjones01**
Here is a take on the CONTRIBUTORS.md based on Data Carpentry set up but for NEON Data Skills. 

* as we still have to hash out structure I've only added the spatio-temporal series.  
* contributors are alphabetized.  That way we don't have to worry about relative contributions (and we won't always be first!) 
* space for all the different relative content types.  Each lesson/ repo writes up just it's own chunk and that gets added to the Data-Skills-Dev CONTRIBUTORS.md when the tutorial/series is copied over (might have to be a manual copy paste?

Questions: 
* can we someone use the author/ reviewers code to automatically update?  Challenges I see are filtering the duplicate names and combining the two fields. 
* will this just live in github or also on the webpage? I added a basic YAML chunk at the top but it is mostly blank.  
* Lot of the old lessons were standalone.  Entry for each one?  Or slowly create series from these? Or "Contributors to other Tutorials" as a catchall?

### CODE REVIEW

**Comment by mjones01** [comment]
@lwasser  Double check the Hackathon participant names for me.  There is at least 1 I didn't recognize -- I pulled them off a Hackathon Attendees list in gDrive.

**Comment by lwasser** [comment]
this looks good! did the DC lessons have a YAML header? i don't think we need it as this page is solely for github. it won't be rendered as a web page.

**Comment by mjones01** [comment]
Perfect.  I'll revise.  I wasn't sure if we wanted this to build or not.

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+## NEON Science Graphics
+The graphics used throughout the NEON Data Skills portal, unless specifically 
+attributed, have been created by NEON staff with the majority attributed to 
+Colin Williams. 
+
+***
+
+## Teaching Data Subsets
+
+### Spatio-temporal Series Teaching Data Subsets
```
Comment (lwasser) [inline CONTRIBUTORS.md]: is there a link to the figshare repo here? we should add one if not.

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+categories: 
+
+permalink:
+image:
+  feature: hierarchy_folder.png
+  credit: 
+  creditlink: 
+---
+
+Materials hosted on the NEON Data Skills portal have been developed and adapted
```
Comment (lwasser) [inline CONTRIBUTORS.md]: Materials hosted on the NEON Data Skills portal are developed by NEON staff in collaboration with members of the scientific community.

File: CONTRIBUTORS.md (hunk 1 above)
Comment (mjones01) [inline CONTRIBUTORS.md]: done.

File: CONTRIBUTORS.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+The original materials were developed during a lesson building hack-a-thon 
+jointly organized by Leah Wasser (NEON), Tracy Teal (Data Carpentry), Mike 
+Smorul (National Socio-Environmental Synthesis Center (SESYNC) and Jason William
+(iPlant Collaborative). 
+
+Hack-a-thon participants included Mike Alonzo, 
+Sean Barberie, Ben Best, Zack Bryn, Jonah Duckles, Marisa Guarinello, 
+Jeff Hollister, Megan A. Jones, Matthew Kwit, Kristina Riemer, Dave Roberts, 
+Keely Roth, Mike Smorul, Courtney Soderberg, Joseph Stachelek, 
+Kaitlin Stack Whitney, Tracy Teal, Leah A. Wasser, Jason Williams, and
```
Comment (mjones01) [inline CONTRIBUTORS.md]: Keely Roth is the name I don't remember.

File: CONTRIBUTORS.md (hunk 2 above)
Comment (mjones01) [inline CONTRIBUTORS.md]: Done

File: CONTRIBUTORS.md (hunk 3 above)
Comment (lwasser) [inline CONTRIBUTORS.md]: Keely was there! She worked with Mike and Jonah in the remote sensing group .

**Comment by lwasser** [comment]
It looks great. Thank you!
Leah

**Comment by lwasser** [comment]
Hey @mjones01 thank you for getting this going. have a look at the other repos as we need to fill out the readme as well. The readme welcomes folks to the repo - explains what it is. And we can direct them to 

1. the contributors and contributing files, etc. 

maybe we should talk about this because we don't necessarily need all three? have a look and see what you think. We could potentially merged ALL three into the readme in fact.  i'm going to hold off on merging this until we decide what we'd like to do.

**Comment by mjones01** [comment]
This PR now has:

* processing directory & files for Boulder Flood video/lessons
* Example Post, YAML post, and associated image file from Template Repo.

**Pull request merged**